packages/expo-file-system/ios/FileSystemModule.swift

```diff
@@ -256,7 +256,15 @@ public final class FileSystemModule: Module {
 
     AsyncFunction("getFreeDiskStorageAsync") { () -> Int64 in
     // Uses required reason API based on the following reason: E174.1 85F4.1
-      let resourceValues = try getResourceValues(from: documentDirectory, forKeys: [.volumeAvailableCapacityForImportantUsageKey])
+      var keyToQuery: URLResourceKey {
+#if os(tvOS)
+        return .volumeAvailableCapacity
+#else
+        return .volumeAvailableCapacityForImportantUsageKey
+#endif
```

Comment:
My personal preference is to order such directives starting from the platforms that we care the most and are our primary focus (iOS), but it's up to you - I don't mind leaving it as is 😉 

```suggestion
#if !os(tvOS)
        return .volumeAvailableCapacityForImportantUsageKey
#else
        return .volumeAvailableCapacity
#endif
```